luxonis_ml/nn_archive/config_building_blocks/base_models/input.py

```diff
@@ -36,12 +38,18 @@ class PreprocessingBlock(BaseModelExtraForbid):
         description="Standardization values in channel order. Typically, this is BGR order.",
     )
     reverse_channels: Optional[bool] = Field(
-        None,
-        description="If True, color channels are reversed (e.g. BGR to RGB or vice versa).",
+        True,
+        deprecated="Deprecated, use `dai_type` instead.",
+        description="If True input to the model is RGB else BGR.",
     )
     interleaved_to_planar: Optional[bool] = Field(
-        None,
-        description="If True, format is changed from interleaved to planar.",
+        False,
+        deprecated="Deprecated, use `dai_type` instead.",
+        description="If True input to the model is interleaved (NHWC) else planar (NCHW).",
+    )
+    dai_type: str = Field(
```

Comment:
We should probably validate the value

Reply:
+1 on validating this str value. Maybe have a list of all the possible data types from DAI and ensure the given value exists there?